packages/forma-36-react-components/src/components/Tooltip/Tooltip.tsx

```diff
@@ -78,7 +80,7 @@ interface ArrowPositionState {
 export const Tooltip = ({
   children,
   className,
-  containerElement: ContainerElement,
+  containerElement: ContainerElement = 'span',
```

Comment:
Why do we need to declare the default of this prop again?

Reply:
Good point, no. I'll remove it from `defaultProps`

Reply:
when we declare it only in `defaultProps` typescript complains

![Screenshot 2020-10-01 at 10 02 16](https://user-images.githubusercontent.com/6163988/94783594-322e3480-03cd-11eb-9299-56956aef05ab.png)

Reply:
I see! I actually prefer declaring the way you did hahaha 
But I replicate the Component.defaultProps pattern that I saw around the codebase

Reply:
@gui-santos do we keep it in `defaultProps` so we can keep the storybook control, or no need 😬?

Reply:
I think storybook will understand the defaults if you pass them the way you did.... but I'm not sure. I'm 80% sure hahaha

Reply:
test it! If it explodes them we keep them in both places 🤷

Reply:
Hahaha, your 80% was quite enough, it worked, storybook translated it into a control, but didn't pre-fill it with `span`, I removed it anyway 👍